openmetadata-service/src/main/java/org/openmetadata/service/Entity.java

```diff
@@ -255,11 +255,14 @@ public static void initializeRepositories(Jdbi jdbi) {
           clz.getDeclaredConstructor().newInstance();
         } catch (Exception e) {
           try {
-            clz.getDeclaredConstructor(Jdbi.class).newInstance(jdbi);
+            clz.getDeclaredConstructor(OpenMetadataApplicationConfig.class).newInstance(config);
```

Comment:
why are we changing this? why do we need to pass Config to repositories?

Reply:
Hey @harsha , need it to build OpenMetadataServerConnectionBuilder for insights pipeline , we could take it as input for App init () method , but then not every app needs OpenmetadataApplicationConfig, let me know if you need me to update it to pass through init method of app

Reply:
ok got it, we need do something better here like switch case statement, but we can do it later